platform/android/os_android.cpp

```diff
@@ -768,4 +739,5 @@ OS_Android::OS_Android(GFXInitFunc p_gfx_init_func, void *p_gfx_init_ud, OpenURI
 }
 
 OS_Android::~OS_Android() {
+	delete godot_java;
```

Comment:
Doesn't seem this should be the location for it since the class doesn't own the passed GodotJavaWrapper pointer.

Reply:
Freeing the GodotJavaWrapper (and GodotIOJavaWrapper) instance might be more appropriate when the Godot.java activity is going through its onDestroy lifecycle callback.
There doesn't seem to be a corresponding function to relay the call in GodotLib.java (like there's for onCreate with GodotLib#initialize) so that may need to be added as well.

Reply:
Making it so :)